lib/moab/storage_object_validator.rb

```diff
@@ -158,11 +156,7 @@ def check_optional_content_dir(version_path, allow_content_subdirs = true)
     end
 
     def contains_forbidden_content_sub_dir?(path)
-      sub_dirs(path).detect { |sub_dir| content_sub_dir_forbidden?(sub_dir) }
-    end
-
-    def content_sub_dir_forbidden?(dirname)
-      FORBIDDEN_CONTENT_SUB_DIRS.include?(dirname) || version_dir_format?(dirname)
+      sub_dirs(path).detect { |sub_dir| version_dir_format?(sub_dir) }
```

Comment:
Now only vNNNN directories will be forbidden.

Reply:
Why prohibit those directory names?

Reply:
@andrewjbtw It doesn't seem to be in the Moab "spec" (which is more of a descriptive paper than a normative spec, AFAICT), so... I don't know. Any reason to hang on to this, @andrewjbtw @sul-dlss/infrastructure-team @ndushay ?

Reply:
I think it might be vestigial from our first storage root before ceph having pre-moabs in a diff form.  @julianmorley ?